### PR TITLE
Clean up api

### PR DESCRIPTION
**Pull request by christophsax**
Removed prefix from most functions, limit exported functions to a few.

### CODE REVIEW

**Comment by christophsax** [comment]
That's it for the moment - no additional PRs

File: R/path.R [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#' Show Exchange Path of TV
+#'
+#' The main R session uses this path to exchange information with the TV session.
+#' Exported for debugging.
```
Comment (krlmlr) [inline R/path.R]: For debugging we can use `tv:::path()` ?

Maybe it's sufficient to document with `@keywords internal` ?

File: NAMESPACE [hunk 2]
```diff
@@ -3,10 +3,8 @@
 S3method(print_html,default)
 export(off)
 export(on)
+export(path)
 export(print_html)
-export(pull_obj)
-export(push_obj)
-export(tv_remote)
-export(tv_remote_own_session)
+export(status)
```
Comment (krlmlr) [inline NAMESPACE]: `get_status()` or `tv_get_status()`? Should all functions use a prefix, or do we expect the users to never attach the package?

File: R/path.R (hunk 1 above)
Comment (christophsax) [reply R/path.R]: true, changed it.

File: NAMESPACE (hunk 2 above)
Comment (christophsax) [reply NAMESPACE]: No prefix for anything. The only exported functions are now `on()`, `off()`, and `status()`, all of them usually used without attaching.

I feel that short `status()` function sound nicer together with `on()` and `off()`. Nicer than `get_status()`, and tells the same.

**Pull request merged**